tests/NClient/NClient.Standalone.Tests/NClient.Standalone.Tests.csproj

```diff
@@ -23,6 +23,7 @@
         <ProjectReference Include="..\..\..\src\NClient.Providers\NClient.Providers.Transport.SystemNetHttp\NClient.Providers.Transport.SystemNetHttp.csproj" />
         <ProjectReference Include="..\..\..\src\NClient.Providers\NClient.Providers.Serialization.SystemTextJson\NClient.Providers.Serialization.SystemTextJson.csproj" />
         <ProjectReference Include="..\..\..\src\NClient\NClient.Standalone\NClient.Standalone.csproj" />
+        <ProjectReference Include="..\..\..\src\NClient\NClient\NClient.csproj" />
```

Comment:
Ideally, the assembly should not depend on `NClient.csproj` because it breaks the hierarchy. As far as I can see, we can remove this link and nothing will break